Makefile

```
@@ -111,6 +112,9 @@ gen-cli-doc: ## Generate CLI documentation
 gen-changelog: ## Generate CHANGELOG.md and migration guide updates
 	./hack/generate/changelog/gen-changelog.sh
 
+gen-olm-bindata:
```

We really don't need to add the `gen-` prefix to these. Typically you make the rule name the thing you're "making"
```suggestion
bindata:
```